domain/src/main/java/com/distillery/android/domain/Dependencies.kt

```diff
@@ -0,0 +1,8 @@
+package com.distillery.android.domain
+
+import android.content.Context
+
+interface Dependencies {
+
+    fun loadDependencies(androidContext: Context)
```

Comment:
There's no need to specify that it is an android context, we don't have any other. In case it should be instance of activity - then better to narrow down the argument type to activity.
```suggestion
    fun loadDependencies(content: Context)
```